### PR TITLE
EO-658 UX Design Feedback on Signals V2 Pages

### PR DESCRIPTION
**Pull request by 21042jim**
### What Changed
See A/C -> https://jira.int.messagesystems.com/browse/EO-658

### How To Test
- Log into appteam or add the `feature_signals_v2` UI option to your account.
- Visit an overview and details pages

### To Do
- [ ] Get sentence in the health score chart header that explains the reference lines

### CODE REVIEW

File: src/pages/signals/HealthScorePage.js [hunk 1]
```diff
@@ -96,9 +96,10 @@ export class HealthScorePage extends Component {
                   selected={selectedDate}
                   hovered={hoveredDate}
                   timeSeries={data}
-                  tooltipContent={({ payload = {}}) => (
-                    <TooltipMetric label='Health Score' value={`${roundToPlaces(payload.health_score * 100, 1)}`} />
-                  )}
+                  tooltipContent={({ payload = {}}) => {
+                    const fillColor = (payload.ranking) ? thresholds[payload.ranking].color : '#63666A';
```
Comment (jonambas) [inline src/pages/signals/HealthScorePage.js]: `color` has a default value in `TooltipMetric`, no need to pass the second hex through

File: src/pages/signals/dashboards/components/HealthScoreChart/HealthScoreChart.js [hunk 2]
```diff
@@ -112,29 +120,34 @@ export function HealthScoreChart(props) {
               selected={selectedDate}
               hovered={hoveredDate}
               timeSeries={accountData.history}
-              tooltipContent={({ payload = {}}) => (
-                <TooltipMetric label='Health Score' value={`${roundToPlaces(payload.health_score, 1)}`} />
-              )}
+              tooltipContent={({ payload = {}}) => {
+                const fillColor = (payload.ranking) ? thresholds[payload.ranking].color : '#63666A';
```
Comment (jonambas) [inline src/pages/signals/dashboards/components/HealthScoreChart/HealthScoreChart.js]: Same here, don't need to pass a default

File: src/pages/signals/dashboards/components/HealthScoreChart/HealthScoreChart.js [hunk 3]
```diff
@@ -112,29 +120,34 @@ export function HealthScoreChart(props) {
...
               yAxisRefLines={[
-                { y: 80, stroke: 'green', strokeWidth: 1 },
-                { y: 55, stroke: 'red', strokeWidth: 1 }
+                { y: 80, stroke: thresholds.good.color, strokeWidth: 1 },
+                { y: 55, stroke: thresholds.danger.color, strokeWidth: 1 }
               ]}
               yKey='health_score'
               xAxisProps={getXAxisProps()}
-              yAxisProps={{ ticks: [0,20,40,55,80,100]}}
+              yAxisProps={{ ticks: [0,55,80,100]}}
```
Comment (jonambas) [inline src/pages/signals/dashboards/components/HealthScoreChart/HealthScoreChart.js]: Will need to make the same change in HealthScorePage too

**Comment by jonambas** [comment]
Also in `HealthScoreChart.module.scss`

Can we change the divider css

```css
.Divider {
- border: 1px solid color(gray, 6);
  margin-top: rem(10);
  height: 60%;
  margin-right: spacing();
+ width: 1px;
+ background: color(gray, 6);
}
```

**Comment by jonambas** [comment]
One more tiny change, this line in health score thresholds
```js
- description: <>A Health score above 80 is considered to be <strong>very good</strong>.</>
+ description: <>A Health score above 80 is considered to be <strong>good</strong>.</>
```

File: src/pages/signals/tests/__snapshots__/HealthScorePage.test.js.snap [hunk 4]
```diff
@@ -14,21 +14,9 @@ exports[`Signals Health Score Page bar chart props renders tooltip content for c
...
 `;
 
-exports[`Signals Health Score Page bar chart props renders tooltip content for health score 1`] = `
-<TooltipMetric
-  color="#63666A"
-  label="Health Score"
-  value="12"
-/>
-`;
+exports[`Signals Health Score Page bar chart props renders tooltip content for health score 1`] = `""`;
```
Comment (jonambas) [inline src/pages/signals/tests/__snapshots__/HealthScorePage.test.js.snap]: Hm these should still render

Comment (jonambas) [reply src/pages/signals/tests/__snapshots__/HealthScorePage.test.js.snap]: oh the `payload` prop in the tests dont have ranking

**Comment by 21042jim** [comment]
> Lemme ask for that copy

Any luck?

**Pull request merged**